Proofreading-OV-Runtime (#11658)
openvinotoolkit/openvino: docs/optimization_guide/model_optimization_guide.md

>  Model optimization is an optional offline step of improving final model performance by applying special optimization methods like quantization, pruning, preprocessing optimization, etc. OpenVINO provides several tools to optimize models at different steps of model development:

Author: ```suggestion
 Model optimization is an optional offline step of improving final model performance by applying special optimization methods, such as quantization, pruning, preprocessing optimization, etc. OpenVINO provides several tools to optimize models at different steps of model development:
```

![](../img/DEVELOPMENT_FLOW_V3_crunch.svg)

> Below diagram was prepared to help you undertstand, which development optimization tool you need to use: 

Author: ```suggestion
The diagram below will help you understand, which development optimization tool you need to use: 
```

![](../img/WHAT_TO_USE.svg)

> Post-training methods are limited in terms of achievable accuracy. For challenging use cases accuracy might degrade. In that case, training-time optimization with NNCF is an option.

Author: ```suggestion
Post-training methods are limited in terms of achievable accuracy. For challenging use cases, accuracy might degrade. When it happens, training-time optimization with NNCF is an option.
```
